### DISCUSSION TITLE
support hls proxy mode

### DISCUSSION THREAD

**Discussion by zhaibin18** [bluenviron/mediamtx#326] [General]
## Which version are you using?

v0.14.2

## Enhancement request

support  hls proxy mode with the config:

paths:
  proxied1:
    source: http://url1/stream1.m3u

  proxied2:
    source: http://url2/stream2.m3u

**Comment by aler9** [comment]
Yes, this is a good enhancement, adding it to the list

**Reply by vitorcampea** [reply]
good feature

**Reply by aler9** [reply]
HLS support is coming - reading with HLS is supported in main, and proxying HLS is coming soon.

**Comment by aler9** [comment]
This feature is now available in v0.17.3

[a comment by an automated account is not shown]
